.gitlab-ci.yml

```diff
@@ -0,0 +1,62 @@
+include: [component: $CI_SERVER_FQDN/lc-components/id_tokens/id_tokens-component@main]
+      
+build-job:
+  tags:
+    - oslic
+    - shell
+  script:
+    - echo "Starting build stage for mpifileutils"
+  stage: build
+  script:
+    - pwd
+    - module load openmpi-gnu
```

Comment:
It might be good to add `module --force purge` before loading openmpi-gnu to ensure you don't inherit unexpected modules from the environment

Reply:
done.